src/main/c/init.c

```diff
@@ -0,0 +1,29 @@
+#include "init.h"
+
+jclass peStatClass;
```

Comment:
Don't store this -- you only need it to get the field IDs.  I wonder if you need to call DeleteLocalRef on it when you're done with it?  Not sure.